v0/destinations/hs/util.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-await-in-loop */
```

Comment:
This is needed for processing paginated response when searching hubspot. we can't avoid await in loop as response to the request contains the pagination details